tools/cli_commands/erv2.py

```diff
@@ -185,10 +185,10 @@ def build_cdktf(self, credentials: Path) -> None:
                         "run",
                         "--name",
                         "erv2",
-                        "-v",
-                        f"{input_file!s}:/inputs/input.json:Z",
-                        "-v",
-                        f"{credentials!s}:/credentials:Z",
+                        "--mount",
+                        f"type=bind,src={input_file!s},dst=/inputs/input.json",
+                        "--mount",
+                        f"type=bind,src={credentials!s},dst=/credentials",
```

Comment:
This won't work on Linux with SELinux enabled :( My 1st version was using `--mount` instead of `-v`

Reply:
😞 then I have to update other docs using `--mount` to `-v`...

Reply:
removed that commit